src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -537,3 +548,29 @@ def publish_cluster_nodes(self, node_count: int):
             self.handle_supabase_error()
 
         logging.info(f"cluster nodes: {UPDATE_CLUSTER_NODE_COUNT} => {data}")
+
+    def persist_events_block(self, block: EventsBlock, finding_id):
+        db_events = []
+        for row in block.render_rows():
+            event = dict(zip(block.headers, row))
```

Comment:
This is a little bug prone
If someone accidentally add another column to table it will fail.
I think it's better to use some class that matches the row
It's also hard to understand what are the DB columns